cfgov/unprocessed/apps/youth-employment-success/js/yes-store.js

```diff
@@ -0,0 +1,17 @@
+import Store from '../../../js/organisms/MortgagePerformanceTrends/stores/store';
```

Comment:
Hmm okay for now, but maybe we should move this store to a `utils` folder that both this app and MortgagePerformanceTrends use.

Reply:
I agree, in a follow up PR I will move this to a shared `utils` folder.